jme3-core/src/main/java/com/jme3/input/ChaseCamera.java

```diff
@@ -547,6 +547,8 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
         if (!enabled) {

```

Comment:
This code would be more readable if you inverted the test.

Reply:
I agree, I did not want to modify too much of the existing code. I can easily make that change though.

Reply:
In general, keeping PRs simple is a good thing. However, the resulting source code will be read 100x times more than the PR will, so modifying 4 lines (instead of 2) isn't "too much".